console.rb

```diff
@@ -12,6 +12,80 @@
 # puts "The first apartment is at #{Apartment.first.address}."
 # puts "It has  #{Apartment.first.tenants.count} tenants."
 
-binding.pry
+# binding.pry
 
-puts "ignore this line, it's just here so the binding.pry above works"
+choice = nil
+while choice != 'q'
+
+  2.times {puts "*" * 70}
```

Comment:
Suggest extracting something like this into a named method to clarify what it does (and since you re-use it later on)